rosdep/python.yaml

```diff
@@ -5969,6 +5969,10 @@ python3-matplotlib:
     '7': null
   slackware: [python3-matplotlib]
   ubuntu: [python3-matplotlib]
+python3-mechanize:
+  debian: [python3-mechanize]
+  fedora: [python3-mechanize]
+  ubuntu: [python3-mechanize]
```

Comment:
Same here; it looks like `python3-mechanize` is only available since focal, but we still support xenial and bionic.  So I'll suggest:

```
ubuntu:
  '*': [python3-mechanize]
  bionic: null
  xenial: null
```